addons/account_payment/tests/test_payment_refund_wizard.py

```diff
@@ -0,0 +1,35 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account_payment.tests.common import AccountPaymentCommon
+from odoo.tests import tagged
+
+
+@tagged('-at_install', 'post_install')
+class TestPaymentRefundWizard(AccountPaymentCommon):
+
+    def setUp(self):
+        super().setUp()
+        self.admin_user = self.env.ref('base.user_admin')
+        self.demo_user = self.env['res.users'].search([('login', '=', 'demo')])
+
+    def test_amount_available_for_refund_consistency(self):
+        self.provider.support_refund = 'full_only'
+        tx = self._create_transaction('redirect', state='done')
+        tx._reconcile_after_done()
+        payment = tx.payment_id
+
+        wizard_admin = self.env['payment.refund.wizard'].with_user(self.admin_user).create({
+            'payment_id': payment.id,
+        })
+        admin_amount = wizard_admin.amount_available_for_refund
+        
+        # Invalidate the cache to simulate a page reload and force a fresh computation 
```

Comment:
```suggestion

        # Invalidate the cache to simulate a page reload and force a fresh computation
```
You should be able to configure your editor in a way that prevents this trailing whitespace issue.